ocs_ci/ocs/constants.py

```diff
@@ -83,6 +83,10 @@
     TEMPLATE_CSI_RBD_DIR, "pod.yaml"
 )
 
+CSI_CEPHFS_POD_YAML = os.path.join(
```

Comment:
Not needed now, it's already there in constants.py

Reply:
@sidhant-agrawal can't find it in constants.py

Reply:
sorry for the confusion. It's not there yet.
shreekar's  PR is having that which is still not merged